docs/brainstorming/consolidation-strategy.md

```diff
@@ -0,0 +1,275 @@
+# Consolidation Strategy: neuroimaging-go-brrrr
+
+> **Status**: Draft / Open for Discussion
+> **Date**: 2024-12-05
+> **Contributors**: Open to all
+
+## The Problem
+
+**HuggingFace has been slow to integrate neuroimaging and BIDS support.** The `datasets` library has excellent support for text, images, and audio, but scientific imaging formats (NIfTI, DICOM, BIDS) have lagged behind.
+
+The Hugging Science community is working together to accelerate this. This document outlines how we can consolidate our efforts under `neuroimaging-go-brrrr` to reduce fragmentation and ship faster.
+
+## Current State of HuggingFace Neuroimaging Support
+
+### What's Already Merged (thanks to community contributions)
+
+| PR | Title | Status | Date |
+|----|-------|--------|------|
+| [#7815](https://github.com/huggingface/datasets/pull/7815) | Add Nifti support | MERGED | Oct 24, 2025 |
+| [#7853](https://github.com/huggingface/datasets/pull/7853) | Fix embed storage nifti | MERGED | Nov 6, 2025 |
+| [#7874](https://github.com/huggingface/datasets/pull/7874) | Nifti visualization support | MERGED | Nov 19, 2025 |
+| [#7878](https://github.com/huggingface/datasets/pull/7878) | Replace papaya with NiiVue | MERGED | Nov 21, 2025 |
+
+> **Note**: While NIfTI support is merged in the `datasets` library, the HuggingFace Hub dataset viewer may lag behind or require dataset regeneration to fully support the new feature type.
```

Comment:
Yep, to me it is not clear if we could do something here. Maybe someone wants to take a look into https://github.com/huggingface/dataset-viewer, but with a brief look I couldn't find any way to tackle this, haven't spend a lot of time on this though